staging/src/k8s.io/kube-scheduler/config/v1/types.go

```diff
@@ -96,6 +96,31 @@ type KubeSchedulerConfiguration struct {
 	// failover with the benefit of lower memory overhead while waiting to become leader.
 	// Defaults to false.
 	DelayCacheUntilActive bool `json:"delayCacheUntilActive,omitempty"`
+
+	// Metric defines the configuration of metrics.
+	// +optional
+	Metric KubeSchedulerMetricConfiguration `json:"metric,omitempty"`
+}
+
+type KubeSchedulerMetricConfiguration struct {
+	// Percentage of plugin metrics to sample.
+	// A value of 100 means all metrics are recorded; lower values enable probabilistic sampling.
+	// Defaults to 10
```

Comment:
```suggestion
	// SamplingRatePercent controls how often the scheduler tries to record metrics that might be too frequent if we record always.
	// It only impacts a plugin_execution_duration_seconds metric for now.
	// Note that setting a bigger value would allow you to record more metric values, but with the trade-off of slower scheduling latency.
	// Defaults to 10
```